pkg/flags/flag.go

```diff
@@ -118,13 +127,18 @@ func verifyEnv(prefix string, usedEnvKey, alreadySet map[string]bool) {
 			continue
 		}
 		if alreadySet[kv[0]] {
-			plog.Infof("recognized environment variable %s, but unused: shadowed by corresponding flag ", kv[0])
```

Comment:
should this be staged as a deprecated warning in 3.3 and then finally fully error out in 3.4?